perception/map_based_prediction/src/path_generator.cpp

```diff
@@ -162,7 +162,7 @@ PredictedPath PathGenerator::generateStraightPath(const TrackedObject & object)
 {
   const auto & object_pose = object.kinematics.pose_with_covariance.pose;
   const auto & object_twist = object.kinematics.twist_with_covariance.twist;
-  const double ep = 0.001;
+  constexpr double ep = 0.001;
   const double duration = time_horizon_ + ep;
```

Comment:
```suggestion
  constexpr double ep = 0.001;
  constexpr double duration = time_horizon_ + ep;
```

Reply:
@kyoichi-sugahara thanks for the suggestion, time_horizon_ cannot be decided at compile time so duration cannot be constexpr

Reply:
oh sorry